commands/plugin_web.go

```diff
@@ -184,7 +185,7 @@ func HandleCommands(w http.ResponseWriter, r *http.Request) (web.TemplateData, e
 func HandlePostCommands(w http.ResponseWriter, r *http.Request) (web.TemplateData, error) {
 	ctx := r.Context()
 	activeGuild, templateData := web.GetBaseCPContextData(ctx)
-	newPrefix := r.FormValue("Prefix")
+	newPrefix := strings.TrimLeft(r.FormValue("Prefix"))
```

Comment:
`strings.TrimLeft` does not work in the same way as `strings.TrimSpace`.
Just to give you an idea of what to do (you need to import `unicode` if it isn't already):
```suggestion
	newPrefix := strings.TrimLeftFunc(r.FormValue("Prefix"), unicode.IsSpace)
```

Reply:
Just saw it as well, thanks though :')

Reply:
`unicode` needs to be imported as stated in the initial review -- would suggest that you do this from an editor instead of via GitHub UI since `gofmt` is probably going to have some ideas as to where to put it. I know you've probably seen this already but just wanted to make it known here in case Jonas comes back and merges before you've edited :-)